File: apps/mobile-portfolio/src/domain/entities/customer.ts
export interface Customer {
  customer_id: string;
  name: string;
  email: string | null;
  kyc_status: string | null;
  created_at: string;
  ai_identity_score?: number | null;
}

export interface CustomerCreate {
  name: string;
  email?: string | null;
  kyc_status?: string | null;
}

File: apps/server-go/internal/domain/customer.go
package domain

import "time"

type Customer struct {
	CustomerID string    `json:"customer_id"`
	Name       string    `json:"name"`
	Email      *string   `json:"email"`
	KYCStatus  *string   `json:"kyc_status,omitempty"`
	CreatedAt  time.Time `json:"created_at"`
}
